content/millennium/r4/clinical/care-provision/care-team.md

```diff
@@ -75,11 +75,11 @@ Notes:
 * The `category` parameter
   * Only supports the codes longitudinal and encounter.
   * Can only be used with the `patient` parameter.
-  * The longitudinal and encounter codes are defined by the CareTeam category system
+  * The longitudinal and encounter codes are defined by the CareTeam category system.
 
-* The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
-
-* The `_revinclude` parameter may be provided with the `_id/patient` parameter. Example: `_id=LIFETIME_PROVIDER-4169494-0-4105597-0-0-0&_revinclude=Provenance:target`
+* The `_revinclude` parameter 
+  * May be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
+  * May be provided with the `_id/patient` parameter. Example: `_id=LIFETIME_PROVIDER-4169494-0-4105597-0-0-0&_revinclude=Provenance:target`
```

Comment:
similar to CarePlan, let's split _id and patient here.

Reply:
Done
![image](https://github.com/cerner/fhir.cerner.com/assets/144141792/b1a25258-acc1-4d71-a596-f444bd7edfff)